catalog/app/containers/Bucket/Summary.js

```diff
@@ -30,7 +30,7 @@ import * as requests from './requests'
 
 const README_RE = /^readme\.md$/i
 const SUMMARIZE_RE = /^quilt_summarize\.json$/i
-const IMAGE_EXTS = ['.jpg', '.jpeg', '.png', '.gif']
+const IMAGE_EXTS = ['.jpg', '.jpeg', '.png', '.gif', '.webp', '.bmp']
```

Comment:
Thanks can you add .tif and .tiff?

Reply:
@akarve afaik tif isnt supported by browsers (and by our thumbnailing endpoint as well)

Reply:
oh yeah, let's take this PR and then we'll modify the image thumbnailer to return JPG or PNG when the input is TIF/F